### DISCUSSION TITLE
RuntimeError" could not connect to Sqlite

### DISCUSSION THREAD

**Discussion by houtvm** [spine-tools/SpineOpt.jl#1061] [Support]
### Describe your problem

Hi all,

I'm a newby finally stepping into the Spine Universe :-)

I followed all the steps from the tutorial to run the simple test system. When running SpineOpt it takes a long time to run (and it is not finished yet but that seems to be due to Julia config) and other than that in Command Prompt a RuntimeError is given, stating: 

RuntimeError: Could not connect to 'sqlite:///////C:\Users\houtvm\SpineTools2\.spinetoolbox\items\output\Output.sqlite': (sqlite3.OperationalError) unable to open database file
(Background on this error at: http://sqlalche.me/e/13/e3q8). Please make sure that 'sqlite:///////C:\Users\houtvm\SpineTools2\.spinetoolbox\items\output\Output.sqlite' is a valid sqlalchemy URL.

The link doesn't give a good hint how this can be solved. Does any one of you maybe have an idea? Both urls (input/output) are linked to SpineOpt (see image attachment).

![image](https://github.com/user-attachments/assets/4cf9785f-3753-4741-ae3e-b23f461eb191)

Thanks in advance!

### What versions are you using?

SpineOpt: installed through SpineToolbox, version?
SpineInterface: ?
SpineToolbox: 0.8.2
Julia: 1.10.4

**Comment by soininen** [comment]
Toolbox 0.8.3 has been released which will hopefully fix the bug you have encountered. If you installed Toolbox via `pipx`, you can update with

```
python -m pipx upgrade spinetoolbox
```

Let us know if the issue still persists after update.
